results/pkg/api/server/server_test.go

```diff
@@ -322,17 +342,29 @@ func TestListResults(t *testing.T) {
 		expectStatus codes.Code
 	}{
 		{
-			name:         "test simple query",
+			name:         "test query taskrun",
 			filter:       `taskrun.api_version=="v1beta1"`,
 			expect:       []*pb.Result{gotResults[0], gotResults[2], gotResults[3]},
 			expectStatus: codes.OK,
 		},
 		{
-			name:         "test query with simple function",
+			name:         "test query pipelinerun",
+			filter:       `pipelinerun.api_version=="v1beta1"`,
```

Comment:
Let's also test unions of mixed results - e.g. `taskrun.api_version=="v1beta1" || pipelinerun.api_version=="v1beta1"`

Reply:
Great! Added :)